skills_DS/api/skills_extraction.py

```diff
@@ -0,0 +1,86 @@
+from .models import Location, JobPosting, JobTitle, Skill
+import re
+import math
+import nltk
+
+nltk.download('stopwords')
+nltk.download('punkt')
+nltk.download('averaged_perceptron_tagger')
+nltk.download('wordnet')
+nltk.download('words')
```

Comment:
These should be added in the docker setup script instead.

Reply:
Don't worry about it for now - I will move them to the setup script once I merge the pdf parsing branch.